server/src/resolvers/getVariantsResolver/adapters/g4rdAdapter.ts

```diff
@@ -86,23 +89,55 @@ const getG4rdNodeQuery = async ({
       },
       {
         headers: { Authorization, 'Content-Type': 'application/json', Accept: 'application/json' },
-        httpsAgent: new https.Agent({ rejectUnauthorized: false }),
       }
     );
 
     // Get patients info
     if (G4RDVariantQueryResponse) {
-      const individualIds = G4RDVariantQueryResponse.data.results.map(
-        v => v.individual.individualId
-      );
-      const patientUrl = `${process.env.G4RD_URL}/rest/patients/fetch?${individualIds
-        .map(id => `id=${id}`)
-        .join('&')}`;
-
-      G4RDPatientQueryResponse = await axios.get<G4RDPatientQueryResult>(patientUrl, {
-        headers: { Authorization, 'Content-Type': 'application/json', Accept: 'application/json' },
-        httpsAgent: new https.Agent({ rejectUnauthorized: false }),
-      });
+      let individualIds = G4RDVariantQueryResponse.data.results
+        .map(v => v.individual.individualId!)
+        .filter(Boolean); // Filter out undefined and null values.
+
+      // Get all unique individual Ids.
+      individualIds = [...new Set(individualIds)];
+
+      if (individualIds.length > 0) {
+        const patientUrl = `${process.env.G4RD_URL}/rest/patients/fetch?${individualIds
+          .map(id => `id=${id}`)
+          .join('&')}`;
```

Comment:
Here, you may want to use the URL API, at least URLSearchParams, in case a string individual id has a funny character in it that needs to be URL-escaped.